### ISSUE TITLE
[Bug] Cannot use `:Bdelete` in terminal buffer

### ISSUE THREAD

**Issue by mehalter** [famiu/bufdelete.nvim#28]
**Neovim version**
```
NVIM v0.8.0
Build type: Release
LuaJIT 2.1.0-beta3
Compiled by builduser

Features: +acl +iconv +tui
See ":help feature-compile"

   system vimrc file: "$VIM/sysinit.vim"
  fall-back for $VIM: "/usr/share/nvim"

Run :checkhealth for more info
```

**Describe the bug**

Unable to use `:Bdelete` or `:Bdelete!` in terminal buffer (also applies to the lua API with `require("bufdelete").bufdelete(...)`)

**To Reproduce**
1. Open NeoVim (`nvim`)
2. Open a terminal buffer (`:terminal`)
3. Try to delete the buffer (`:Bdelete` or `:Bdelete!`)

```
Error executing Lua callback: ...te/pack/packer/opt/bufdelete.nvim/lua/bufdelete/init.lua:140: Vim:E89: term://~/.config/nvim/lua//221736:/bin/zsh will be killed (add ! to override)                                                                                     
stack traceback:                                                                                                                                                                                                                                                          
        [C]: in function 'bdelete'                                                                                                                                                                                                                                        
        ...te/pack/packer/opt/bufdelete.nvim/lua/bufdelete/init.lua:140: in function 'buf_kill'                                                                                                                                                                           
        ...te/pack/packer/opt/bufdelete.nvim/lua/bufdelete/init.lua:221: in function 'buf_kill_cmd'                                                                                                                                                                       
        ...te/pack/packer/opt/bufdelete.nvim/lua/bufdelete/init.lua:225: in function <...te/pack/packer/opt/bufdelete.nvim/lua/bufdelete/init.lua:225>    
```

**Expected behavior**

Close the terminal buffer

**Does this error occur in the minimal init file?**
Yes, this error still occurs

**Provide modified minimal_init.lua**
Not applicable, the provided `minimal_init.lua` file is sufficient for replicating this error.

**Screenshots**

![2022-11-01_11:36:53_screenshot](https://user-images.githubusercontent.com/1591837/199273843-7fcefe1c-099b-42a0-8f4d-6d7323863132.png)


**Additional context**

Terminal recording of this bug: https://asciinema.org/a/RmF9dYhGrlmSM7PMkebc21nkw

**Comment by famiu** [comment]
`:Bdelete!` in terminal buffer works fine for me. As for `:Bdelete`, that behavior is intended as it's the same as `:bdelete`

**Comment by mehalter** [comment]
I cannot replicate this behavior you are describing @famiu . As you can see in my recording that I am running `:Bdelete!` using the provided minimal configuration and am unable to close the terminal buffer

**Comment by famiu** [comment]
> I cannot replicate this behavior you are describing @famiu . As you can see in my recording that I am running `:Bdelete!` using the provided minimal configuration and am unable to close the terminal buffer

Ah, it turns out I didn't pull my last two PRs for my config, sorry about that. Should be fixed with #30
